dist/vendor.sh

```diff
@@ -26,8 +26,12 @@ cargo vendor > "${CARGO_CONFIG}"
 # some vendored files aren't world readable which has lead to issues in container builds when trying out the workflow
 find ./vendor -type f -exec chmod a+r {} \;
 
-git add --force "${FILES_TO_COMMIT[@]}"
-git diff --quiet --staged -- "${FILES_TO_COMMIT[@]}" || {
-  echo Files changed, committing...
-  git commit -m "Update vendored dependencies and rh-manifest.txt" -- "${FILES_TO_COMMIT[@]}"
-}
+# check the a variable as we don't want to commit new files on CI
+# it can also be set by developers which want to skip the git operations
+if [ "${OPENSHIFT_CI}" != "true" ]; then
+  git add --force "${FILES_TO_COMMIT[@]}"
```

Comment:
Why `--force` btw?

Reply:
I was bitten by a local gitignore file at one point with this script so this is a guarantee it doesn't happen to anyone else.

Reply:
What if I have a file with credentials saved there? The script tries to be unnecessarily smart. Let remove `git add/commit` part - I'd rather do it manually tbh

Reply:
Since we have another safety net in the build pipeline, which is it will simply fail if these files aren't up to date, I'm fine removing this one.